lib/scoped_search/query_builder.rb

```diff
@@ -485,6 +489,16 @@ def to_sql(builder, definition, &block)
             raise ScopedSearch::QueryNotSupported, "Don't know how to handle this operator node: #{operator.inspect} with #{children.inspect}!"
           end
         end
+
+        private
+
+        def validate_value(field, value)
+          validator = field.validator
+          if field.validator && field.only_explicit
```

Comment:
> I have put a BP in this method and got this stack: [..] The query is `not has last_report and status.enabled = true`

Sorry, which field is your validator set on? I'm also unsure why this is incorrect - isn't it right that the field is validated, if it's one of `last_report` or `status.enabled`?

> there is also autocomplete use case that gets to the validator with non-explicit fields.

What autocomplete use case do you mean? Autocomplete should only work on named fields, not on free text.